lib/Site.js

```diff
@@ -193,6 +208,30 @@ Site.prototype.createPageData = function (config) {
   });
 };
 
+
+/**
+ * Builds the index of paths to be traversed as addressable pages
```

Comment:
"~`Builds`~`Collects` the ~`index of`~ paths to be traversed as addressable pages"